### PR TITLE
Text utilities - list

### PR DESCRIPTION
**Pull request by lomesshh**

### CODE REVIEW

File: documentation.html [hunk 1]
```diff
@@ -592,9 +592,186 @@ <h4 class="component__sub__title">Checkbox</h4>
...
+            <div class="line"></div>
+
             
-            
+             <!-- List -->
+            <div id="Lists" class="list__main">
+                <h1 class="component__name">List</h1> 
+                <p class="component__desc">Lists allow web developers to group a set of related items in lists.</p>           
+                <h4 class="component__sub__title">Unordered List</h4>
+                <p>An unordered list starts with the <b>ul</b> tag. Each list item starts with the <b>li</b> tag.</p>
```
Comment (vinc3nati) [inline documentation.html:668]: use `<strong>` instead of `<b>` as strong is SEO friendly and is interpreted easily and gives more meaning to the code.

**Comment by Aman-Jain1996** [comment]
Hi Lomesh ,
 i have gone through the code you added under this branch and it looks good to go !

**Review by vinc3nati — APPROVED** [review]

**Review by vinc3nati — APPROVED** [review]
You can merge this branch

File: documentation.html (hunk 1 above)
Comment (lomesshh) [reply documentation.html:668]: Done, I replaced all b tag's with the strong tag
Thank You.

**Review by surajmore98 — COMMENTED** [review]
looks good.

**Pull request merged**